organize_content/exercise2-organize-content.md

```diff
@@ -3,21 +3,27 @@
 Git is a distributed version control system, which means you can work locally,
```

Comment:
**Comment** @Preshh0 This applies to the whole of this file. Please split this file into sections and add a heading for each section. That would make it easier for the user to find what they're looking for.